examples/vg-specs/area.vg.json

```diff
@@ -33,6 +33,9 @@
                     ],
                     "fields": [
                         "count"
+                    ],
+                    "as": [
```

Comment:
I was just wondering why this is on this PR. Seems a bit unrelated but whatever.